src/components/Label/Label.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
+
+const StyledLabel = styled.label<LabelProps>``;
+
+type LabelProps = React.LabelHTMLAttributes<HTMLLabelElement> & {
+  input?: string;
+};
+
+const Label = (props: LabelProps) => {
+  console.log(`From Label${props}`);
```

Comment:
This is probably going to be remove later, but a better way would be to use log point in chrome dev tools